internal/authn/oidc/oidc_test.go

```diff
@@ -0,0 +1,209 @@
+package oidc
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"github.com/MicahParks/keyfunc"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/openfga/openfga/internal/authn"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/metadata"
+	"log"
+	"strings"
+	"testing"
+)
+
+func TestRemoteOidcAuthenticator_Authenticate(t *testing.T) {
+	t.Run("When the authorization header is missing from the gRPC metadata of the request, returns 'missing bearer token' error.", func(t *testing.T) {
```

Comment:
Use `_` instead of whitespace please. It makes it easier to search/find since Go replaces whitespace with underscores in `go test` output.